thirdparty/maven/zimbra-maven/debian/watch

```diff
@@ -0,0 +1,2 @@
+version=3
+http://archive.apache.org/dist/maven/maven-3/3.3.9/source/apache-maven-(\d+\.\d+\.\d+).tar.xz
```

Comment:
Added "pvers" instead of hard coded version number, and url changed from "http" to "https".